guides/common/modules/proc_creating-compute-profiles.adoc

```diff
@@ -19,3 +22,67 @@ You can apply compute profiles to all supported compute resources:
 . Click *Submit*.
 A new window opens with the name of the compute profile.
 . In the new window, click the name of each compute resource and edit the attributes you want to set for this compute profile.
+
+[id="cli-creating-compute-profiles_{context}"]
+.CLI procedure
+. Create Compute Profile:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile create --name "_My_Compute_Profile_"
+----
++
+. Set atributes for the Compute Profile using:
```

Comment:
```suggestion
. Set attributes for the compute profile:
```

Reply:
Just to explain this suggestion:

* s/atributes/attributes/ -- typo
* s/using// -- redundant, and also we have a convention to avoid using incomplete sentences in introduction like these (incomplete sentence might cause problems when translating).
* s/Compute Profile/compute profile/ -- the existing content uses lower case (see for example the heading on line 2)

Reply:
Please check the rest of your PR and make sure you're using `compute profile` rather than `Compute Profile` everywhere in your additions.

BTW, don't let line 20 puzzle you:

> . In the {ProjectWebUI}, navigate to *Infrastructure* > *Compute Profiles* and click *Create Compute Profile*.

This one uses `Compute Profile` in upper case formatting and that's okay (sort of) because that's what the labels in the Foreman web UI say.

![Screenshot from 2024-04-30 19-27-39](https://github.com/theforeman/foreman-documentation/assets/14296616/4d0d283d-234d-4f46-9c2f-c4558748f66f)

The documentation writes this differently, which is not ideal, but it's also out of scope of your PR so you can just stick to the documentation conventions in your PR.

Reply:
Thanks for the review.

I was quite unsure if i should add the delete, view details commands. My thought was that those commands are nowhere in the documentation and it would be quite useful for the user to know that those commands exist and how to use them. But i will try to only keep the commands that are the same type as the procedure above as you mentioned. 

That typo was silly mistake, will quickly fix that, same about "using", i was looking through the docs and saw that "Compute Profiles" is used way more than "compute profiles" so thats why i decided for the first option. Looking back i can see the reasons why are which versions used, the reason being the UI reference.

Going to fix it asap and push new version.